installation/qnap.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: documentation
+title: QNAP NAS
+---
+
+:::
+
+# QNAP NAS
+
+[QNAP NAS](https://www.qnap.com/en) is a famous NAS server solution for home and business, allowing the installation of additional packages.
```

Comment:
I wouldn't use "famous". It sounds like a marketing slogan and we don't want endorse any brand in our documentation..

Reply:
Done. But pls note that Synology install instructions also use the word "famous". Maybe you can change that also...